test/features/pageviews/pageviews.js

```diff
@@ -88,6 +92,38 @@ describe('pageviews endpoints', function () {
         });
     });
 
+    it('should not die on bad data in the v column', function () {
+        return preq.post({
+            uri: server.config.globalURL +
+                 fix(insertProjectEndpoint, 'en.wikipedia', '2') +
+                 '/catch'
```

Comment:
This is not inserting not only to the `v` column, but to the `views` column to. `views` is integer, therefor you get tests failing inside the driver code attempting to convert this 'catch' to a number.